modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -43,7 +43,10 @@ def self.date_from_string(string)
     # further validations:
     validate(
       :birth_date_is_in_the_past,
-      :claimant_birth_date_is_in_the_past
+      :claimant_birth_date_is_in_the_past,
+      :required_claimant_data_is_present,
+      :contestable_issue_dates_are_valid_dates,
```

Comment:
Since date format checking is taken care of by the schema, and we have another "is_in_the_past" method, let's rename this to "contestable_issue_dates_are_in_the_past" (or similar) - it's a bit more descriptive to what it's actually checking.

Reply:
On it.